docs/docs-content/integrations/longhorn.md

```diff
@@ -191,6 +196,76 @@ Longhorn documentation. The examples section contains several use cases of how t
 </TabItem>
 </Tabs>
 
+## Troubleshooting
+
+### Longhorn Deployer Stuck in Init State in Airgap Environments
+
+When you use the Longhorn pack in an airgapped environment, the Harbor Edge-Native Config pack is required for your
+cluster to function. However, you must change the image references in the Longhorn pack to use the base URL of the local
+Harbor registry. Otherwise, the cluster will fail to deploy, with the `longhorn-driver-deployer` pod stuck in the `Init`
+state.
+
+#### Debug Steps
+
+1.  Log in to Palette.
+
+2.  From the left **Main Menu**, select **Profiles**.
+
+3.  Select the cluster profile you use to deploy the cluster with Longhorn.
+
+4.  In the pack YAML for the Longhorn pack, replace the base URL of each image referenced in the pack with the Harbor
+    registry.
+
+    For example, the following is a snippet of the Longhorn pack's YAML before and after you replace the base URL.
+
+    <Tabs>
```

Comment:
Might be good to use the highlight feature, just to help draw attention to the proper part of the YAMl.  I caught myself looking at the label, thinking you were doing magic there 🙈